src/libs/elektra/hooks.c

```diff
@@ -138,6 +168,13 @@ static bool isGoptsEnabledByContract (const KeySet * contract)
 	return isEnabled;
 }
 
+static bool isSpecEnabledByConfig (const KeySet * config)
+{
+	// TODO: check for system:/elektra/hook/spec/enabled or system:/elektra/hook/spec/disabled or something else ... TBD
```

Comment:
I don't think we need to solve this before merging this PR. Can be added later at any time.

Reply:
Agreed.